sencha-workspace/SlateAdmin/app/controller/settings/Courses.js

```diff
@@ -0,0 +1,169 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('SlateAdmin.controller.settings.Courses', {
+    extend: 'Ext.app.Controller',
+
+
+    // controller config
+    views: [
+        'settings.courses.Manager',
+        'settings.courses.Form'
```

Comment:
@BillClinton I can't get this branch to load because this view isn't present in it, can you commit it?